Carvana.Fizzbuzz.Tests/UnitTest1.cs

```diff
@@ -1,13 +1,21 @@
 using Xunit;
 
+using Carvana.Fizzbuzz;
 namespace Carvana.Fizzbuzz.Tests
 {
     public class UnitTest1
     {
+        
         [Fact]
-        public void Test1()
+        public void MainTest()
```

Comment:
I'd recommend making the name of your test something more meaningful. Something that communicates the intent of the test to anyone else that might come and look at it.

Picker_NumberIsMultipleOf3_PrintFizz